.github/workflows/publish_pypi.yml

```
@@ -0,0 +1,37 @@
+name: Publish Python distribution to PyPI
+
+on: push
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
```

Probably only want to run on pushes to `master`.

```suggestion
    if: github.ref == 'refs/heads/master'
    steps:
```

---

That's what I was thinking, but this package isn't changing much, so I left it out.